client/my-sites/importer/newsletter/subscribers/paid-subscribers/connect-stripe.tsx

```diff
@@ -1,12 +1,14 @@
 import { getQueryArg, addQueryArgs } from '@wordpress/url';
 import { QueryArgParsed } from '@wordpress/url/build-types/get-query-arg';
+import { toInteger } from 'lodash';
```

Comment:
can we use from lodash? I remember there were some old discussions around this

Reply:
Good call!